### PR TITLE
sample: platform detection

### PR DESCRIPTION
**Pull request by minh-rakuten**
# Description
Update the sample app for Platform detection.

## Links
MINI-1798

# Checklist
- [x] I wrote/updated tests for new/changed code.
- [x] I ran `yarn sdk buildSdk` and `yarn bridge buildBridge` without issues.
- [x] I ran `yarn sample prettify` and `yarn sample build` without issues.

### CODE REVIEW

File: js-miniapp-sample/src/pages/landing.js [hunk 1]
```diff
@@ -22,14 +22,18 @@ const useStyles = makeStyles((theme) => ({
...
 }));
 
 const Landing = () => {
   const classes = useStyles();
   return (
     <GreyCard className={classes.card}>
       <CardContent className={classes.content}>
-        <p>This is a Demo App of Mini App JS SDK</p>
+        <p>Demo Mini App JS SDK</p>
+        <p className={classes.info}>Platform: {MiniApp.getPlatform()}</p>
```
Comment (ts-leojoseph-r) [inline js-miniapp-sample/src/pages/landing.js:36]: 😉 I was thinking to include some big designs for the Home page😛 . But I guess this is ok for Sample app.

**Review by ts-leojoseph-r — APPROVED** [review]
Looks good to me :+1:

**Review by khairul-alam-licon — APPROVED** [review]
👍

File: js-miniapp-sample/src/pages/landing.js (hunk 1 above)
Comment (minh-rakuten) [reply js-miniapp-sample/src/pages/landing.js:36]: That design could come from another PR. Please feel free to update it!

**Pull request merged**